raphtory-graphql/src/model/mod.rs

```diff
@@ -285,43 +285,21 @@ impl Mut {
             .ok_or("graph_nodes not object")?;
         let node_ids = node_map.keys().map(|key| key.as_str()).collect_vec();
 
-        let _new_subgraph = parent_graph.subgraph(node_ids.clone()).materialize()?;
-        _new_subgraph.update_constant_properties([("name", Prop::str(new_graph_name.clone()))])?;
-
-        let new_subgraph = &_new_subgraph.clone().into_persistent().unwrap();
-        let new_subgraph_data = subgraph.subgraph(node_ids).materialize()?;
-
-        // Copy nodes over
-        let new_subgraph_nodes: Vec<_> = new_subgraph_data
```

Comment:
This will break if the current graph has a load of properties on the nodes/edges/graph which the parent doesn't